fetch.js

```diff
@@ -118,6 +117,43 @@ let buildSQLObject = function (m) {
 };
 
 /**
+ * Build a CSV row from a measurement
+ * Header: location, value, unit, parameter, country, city, sourceName, date_utc, date_local, sourceType, mobile, latitude, longitude
+ * @param {object} m measurement object
+ * @return {string} a string representing one row in a CSV
+ */
+let buildCSVObject = function (m) {
+  let row = [
+    m.location,
+    m.value,
+    m.unit,
+    m.parameter,
+    m.country,
+    m.city,
+    m.sourceName,
+    new Date(m.date.utc).toISOString(),
+    m.date.local,
+    m.sourceType,
+    m.mobile
+  ];
+  let latitude = '';
+  let longitude = '';
+
+  // Handle geo
+  if (m.coordinates) {
+    latitude = m.coordinates.latitude || '';
+    longitude = m.coordinates.longitude || '';
+  }
+  row.push(latitude);
+  row.push(longitude);
+
+  // Add double quotes to each field
+  let quotedRow = row.map(item => `"${item}"`);
+
+  return quotedRow.join(',');
```

Comment:
Can we use something from http://csv.adaltas.com/ as we're already using `csv-parse` in the project? Think it'd be overall safer than making the CSV ourselves.